### PR TITLE
fix: incorrect typing of getPhotos between expo and native

### PR DESCRIPTION
**Pull request by santhoshvai**
## 🎨 UI Changes

<details>
<summary>iOS</summary>

<table>
    <thead>
        <tr>
            <td>Before</td>
            <td>After</td>
        </tr>
    </thead>
    <tbody>
        <tr>
            <td>
                
            </td>
            <td>
                
            </td>
        </tr>
    </tbody>
</table>
</details>

<details>
<summary>Android</summary>

<table>
    <thead>
        <tr>
            <td>Before</td>
            <td>After</td>
        </tr>
    </thead>
    <tbody>
        <tr>
            <td>
                
            </td>
            <td>
                
            </td>
        </tr>
    </tbody>
</table>
</details>

## ☑️ Checklist

- [ ] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [ ] PR targets the `develop` branch
- [ ] Documentation is updated
- [ ] New code is tested in main example apps, including all possible scenarios
  - [ ] SampleApp iOS and Android
  - [ ] Expo iOS and Android

### CODE REVIEW

File: package/src/types/types.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ import type { ExtendableGenerics, LiteralStringForUnion } from 'stream-chat';
 export type Asset = {
   duration: number | null;
   filename: string;
-  fileSize: string;
+  fileSize: number | undefined;
```
Comment (santhoshvai) [inline package/src/types/types.ts:6]: `fileSize` is undefined in expo 
and it is always a number

File: package/src/types/types.ts [hunk 2]
```diff
@@ -3,9 +3,8 @@ import type { ExtendableGenerics, LiteralStringForUnion } from 'stream-chat';
...
   height: number;
-  playableDuration: number | null;
```
Comment (santhoshvai) [inline package/src/types/types.ts:8]: no need of this, we can map it to duration in native and have it common between expo and native

File: package/src/types/types.ts (hunk 1 above)
Comment (khushal87) [reply package/src/types/types.ts:6]:
```suggestion
  fileSize?: number;
```

**Review by khushal87 — APPROVED** [review]
The PR looks good. I hope you have tested it on all platforms. Expo android and iOS, Native android and iOS.

**Pull request merged**